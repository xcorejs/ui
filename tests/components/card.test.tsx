import 'jest-styled-components';

import { Card, Img, LoremIpsum, Paragraph } from '@xcorejs/ui';
import React from 'react';
import renderer from 'react-test-renderer';

test('Card component', () => {
  const component = renderer.create(
    <>
      <Card
        title="Card title"
        media={<Img src="http://placekitten.com/500/300" alt="cat" />}
        footer={<Paragraph>Footer</Paragraph>}
      >
        <LoremIpsum units="sentences" />
      </Card>

      <Card
        title="Card title"
        media={<Img src="http://placekitten.com/500/300" alt="cat" />}
        body={<LoremIpsum units="sentences" />}
        footer={<Paragraph>Footer</Paragraph>}
      />

      <Card
        title="Card title"
        tag="Tag"
        media={<Img src="http://placekitten.com/500/300" alt="cat" />}
        body={<LoremIpsum units="sentences" />}
      />

      <Card
        title="Card title"
        tag="Tag"
        media={<Img src="http://placekitten.com/500/300" alt="cat" />}
        body={<LoremIpsum units="sentences" />}
      />

      <Card
        tag="Tag"
        media={<Img src="http://placekitten.com/500/300" alt="cat" />}
        body={<LoremIpsum units="sentences" />}
      />

      <Card
        innerPadding="5rem"
        title="Thick"
        tag="Tag"
        media={<Img src="http://placekitten.com/500/300" alt="cat" />}
        body={<LoremIpsum units="sentences" />}
      />
    </>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
