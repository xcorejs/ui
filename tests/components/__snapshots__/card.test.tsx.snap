// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component 1`] = `
Array [
  .c4 {
  font-size: 2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 3rem;
  font-size: 2rem;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c7 {
  font-size: 1.5rem;
  line-height: 2rem;
  font-size: 1.5rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 0.3rem;
  box-shadow: 0 0.3rem 0.8rem 0 rgba(36,49,70,0.25);
  width: 100%;
  max-width: 30rem;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #fff;
  color: #1E3441;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 3;
  -ms-flex-order: 3;
  order: 3;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 4;
  -ms-flex-order: 4;
  order: 4;
}

.c9 {
  margin: 0;
  font-size: 1.6rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 2rem;
  font-size: 1.6rem;
  color: #1E3441;
}

<div
    className="c0"
    color="text"
    role="group"
    width="100%"
  >
    <div
      className="c1"
    >
      <div
        className="c2"
        order={1}
      >
        <div
          className="c3"
        >
          <span
            className="c4"
            color="text"
            fontFamily="text"
            fontSize="2rem"
          >
            Card title
          </span>
        </div>
      </div>
    </div>
    <div
      className="c5"
      order={2}
    >
      <img
        alt="cat"
        className=""
        src="http://placekitten.com/500/300"
      />
    </div>
    <div
      className="c6"
      order={3}
    >
      <span
        className="c7"
        color="text"
        fontFamily="text"
        fontSize="1.5rem"
      >
        Ad ad ad ad ad.
      </span>
    </div>
    <div
      className="c8"
      order={4}
    >
      <p
        className="c9"
        color="text"
        fontFamily="heading"
        fontSize="1.6rem"
      >
        Footer
      </p>
    </div>
  </div>,
  .c4 {
  font-size: 2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 3rem;
  font-size: 2rem;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c7 {
  font-size: 1.5rem;
  line-height: 2rem;
  font-size: 1.5rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 0.3rem;
  box-shadow: 0 0.3rem 0.8rem 0 rgba(36,49,70,0.25);
  width: 100%;
  max-width: 30rem;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #fff;
  color: #1E3441;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 3;
  -ms-flex-order: 3;
  order: 3;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 4;
  -ms-flex-order: 4;
  order: 4;
}

.c9 {
  margin: 0;
  font-size: 1.6rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 2rem;
  font-size: 1.6rem;
  color: #1E3441;
}

<div
    className="c0"
    color="text"
    role="group"
    width="100%"
  >
    <div
      className="c1"
    >
      <div
        className="c2"
        order={1}
      >
        <div
          className="c3"
        >
          <span
            className="c4"
            color="text"
            fontFamily="text"
            fontSize="2rem"
          >
            Card title
          </span>
        </div>
      </div>
    </div>
    <div
      className="c5"
      order={2}
    >
      <img
        alt="cat"
        className=""
        src="http://placekitten.com/500/300"
      />
    </div>
    <div
      className="c6"
      order={3}
    >
      <span
        className="c7"
        color="text"
        fontFamily="text"
        fontSize="1.5rem"
      >
        Ad ad ad ad ad.
      </span>
    </div>
    <div
      className="c8"
      order={4}
    >
      <p
        className="c9"
        color="text"
        fontFamily="heading"
        fontSize="1.6rem"
      >
        Footer
      </p>
    </div>
  </div>,
  .c4 {
  font-size: 2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 3rem;
  font-size: 2rem;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c8 {
  font-size: 1.5rem;
  line-height: 2rem;
  font-size: 1.5rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 0.3rem;
  box-shadow: 0 0.3rem 0.8rem 0 rgba(36,49,70,0.25);
  width: 100%;
  max-width: 30rem;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #fff;
  color: #1E3441;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 3;
  -ms-flex-order: 3;
  order: 3;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  white-space: nowrap;
  border: 1px solid #455663;
  border-radius: 0.3rem;
  right: 0;
  top: 0;
  position: absolute;
  padding-left: 0.8rem;
  padding-right: 0.8rem;
  margin: 1rem;
  font-size: 1.2rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  line-height: 2rem;
  font-weight: 500;
  font-size: 1.2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 300ms,background 300ms,border-color 300ms;
  transition: color 300ms,background 300ms,border-color 300ms;
  color: #fff;
  background-color: #455663;
}

<div
    className="c0"
    color="text"
    role="group"
    width="100%"
  >
    <div
      className="c1"
    >
      <div
        className="c2"
        order={1}
      >
        <div
          className="c3"
        >
          <span
            className="c4"
            color="text"
            fontFamily="text"
            fontSize="2rem"
          >
            Card title
          </span>
        </div>
      </div>
    </div>
    <div
      className="c5"
      color="background"
      fontFamily="text"
      fontSize="1.2rem"
      fontWeight={500}
    >
      Tag
    </div>
    <div
      className="c6"
      order={2}
    >
      <img
        alt="cat"
        className=""
        src="http://placekitten.com/500/300"
      />
    </div>
    <div
      className="c7"
      order={3}
    >
      <span
        className="c8"
        color="text"
        fontFamily="text"
        fontSize="1.5rem"
      >
        Ad ad ad ad ad.
      </span>
    </div>
  </div>,
  .c4 {
  font-size: 2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 3rem;
  font-size: 2rem;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c8 {
  font-size: 1.5rem;
  line-height: 2rem;
  font-size: 1.5rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 0.3rem;
  box-shadow: 0 0.3rem 0.8rem 0 rgba(36,49,70,0.25);
  width: 100%;
  max-width: 30rem;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #fff;
  color: #1E3441;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 3;
  -ms-flex-order: 3;
  order: 3;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  white-space: nowrap;
  border: 1px solid #455663;
  border-radius: 0.3rem;
  right: 0;
  top: 0;
  position: absolute;
  padding-left: 0.8rem;
  padding-right: 0.8rem;
  margin: 1rem;
  font-size: 1.2rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  line-height: 2rem;
  font-weight: 500;
  font-size: 1.2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 300ms,background 300ms,border-color 300ms;
  transition: color 300ms,background 300ms,border-color 300ms;
  color: #fff;
  background-color: #455663;
}

<div
    className="c0"
    color="text"
    role="group"
    width="100%"
  >
    <div
      className="c1"
    >
      <div
        className="c2"
        order={1}
      >
        <div
          className="c3"
        >
          <span
            className="c4"
            color="text"
            fontFamily="text"
            fontSize="2rem"
          >
            Card title
          </span>
        </div>
      </div>
    </div>
    <div
      className="c5"
      color="background"
      fontFamily="text"
      fontSize="1.2rem"
      fontWeight={500}
    >
      Tag
    </div>
    <div
      className="c6"
      order={2}
    >
      <img
        alt="cat"
        className=""
        src="http://placekitten.com/500/300"
      />
    </div>
    <div
      className="c7"
      order={3}
    >
      <span
        className="c8"
        color="text"
        fontFamily="text"
        fontSize="1.5rem"
      >
        Ad ad ad ad ad.
      </span>
    </div>
  </div>,
  .c4 {
  font-size: 1.5rem;
  line-height: 2rem;
  font-size: 1.5rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 0.3rem;
  box-shadow: 0 0.3rem 0.8rem 0 rgba(36,49,70,0.25);
  width: 100%;
  max-width: 30rem;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #fff;
  color: #1E3441;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-order: 3;
  -ms-flex-order: 3;
  order: 3;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  white-space: nowrap;
  border: 1px solid #455663;
  border-radius: 0.3rem;
  right: 0;
  top: 0;
  position: absolute;
  padding-left: 0.8rem;
  padding-right: 0.8rem;
  margin: 1rem;
  font-size: 1.2rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  line-height: 2rem;
  font-weight: 500;
  font-size: 1.2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 300ms,background 300ms,border-color 300ms;
  transition: color 300ms,background 300ms,border-color 300ms;
  color: #fff;
  background-color: #455663;
}

<div
    className="c0"
    color="text"
    role="group"
    width="100%"
  >
    <div
      className="c1"
      color="background"
      fontFamily="text"
      fontSize="1.2rem"
      fontWeight={500}
    >
      Tag
    </div>
    <div
      className="c2"
      order={2}
    >
      <img
        alt="cat"
        className=""
        src="http://placekitten.com/500/300"
      />
    </div>
    <div
      className="c3"
      order={3}
    >
      <span
        className="c4"
        color="text"
        fontFamily="text"
        fontSize="1.5rem"
      >
        Ad ad ad ad ad.
      </span>
    </div>
  </div>,
  .c4 {
  font-size: 2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  line-height: 3rem;
  font-size: 2rem;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c8 {
  font-size: 1.5rem;
  line-height: 2rem;
  font-size: 1.5rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  color: #1E3441;
  -webkit-transition: color 300ms;
  transition: color 300ms;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 0.3rem;
  box-shadow: 0 0.3rem 0.8rem 0 rgba(36,49,70,0.25);
  width: 100%;
  max-width: 30rem;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background: #fff;
  color: #1E3441;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: 2;
  -ms-flex-order: 2;
  order: 2;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 5rem;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 5rem;
  -webkit-order: 3;
  -ms-flex-order: 3;
  order: 3;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  white-space: nowrap;
  border: 1px solid #455663;
  border-radius: 0.3rem;
  right: 0;
  top: 0;
  position: absolute;
  padding-left: 0.8rem;
  padding-right: 0.8rem;
  margin: 1rem;
  font-size: 1.2rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  line-height: 2rem;
  font-weight: 500;
  font-size: 1.2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 300ms,background 300ms,border-color 300ms;
  transition: color 300ms,background 300ms,border-color 300ms;
  color: #fff;
  background-color: #455663;
}

<div
    className="c0"
    color="text"
    role="group"
    width="100%"
  >
    <div
      className="c1"
    >
      <div
        className="c2"
        order={1}
      >
        <div
          className="c3"
        >
          <span
            className="c4"
            color="text"
            fontFamily="text"
            fontSize="2rem"
          >
            Thick
          </span>
        </div>
      </div>
    </div>
    <div
      className="c5"
      color="background"
      fontFamily="text"
      fontSize="1.2rem"
      fontWeight={500}
    >
      Tag
    </div>
    <div
      className="c6"
      order={2}
    >
      <img
        alt="cat"
        className=""
        src="http://placekitten.com/500/300"
      />
    </div>
    <div
      className="c7"
      order={3}
    >
      <span
        className="c8"
        color="text"
        fontFamily="text"
        fontSize="1.5rem"
      >
        Ad ad ad ad ad.
      </span>
    </div>
  </div>,
]
`;
